project/Dependencies.scala

```diff
@@ -24,7 +24,6 @@ object Dependencies {
     // when updating config version, update links ActorSystem ScalaDoc to link to the updated version
     val config        = "com.typesafe"                % "config"                       % "1.2.1"       // ApacheV2
     val netty         = "io.netty"                    % "netty"                        % "3.8.0.Final" // ApacheV2
-    val protobuf      = "com.google.protobuf"         % "protobuf-java"                % "2.5.0"       // New BSD
```

Comment:
The question was whether source and binary compatibility is perhaps enough, because build compatibility means that everyone who wants to profit from this change will need to manually exclude a dependency, something that not everybody does every day. But you might right that “drop-in replacement” does bind our hands here.